project_resource_calendar/models/calendar_event.py

```diff
@@ -83,3 +83,7 @@ def _check_room_id_double_book(self):
                         'with any overlapping meetings or events.',
                     )
                 )
+
+    @api.multi
```

Comment:
Why multi you will only print one report for one record

Reply:
@rimbendhaou Done